src/main/java/nurseybook/logic/parser/AddTagCommandParser.java

```diff
@@ -23,14 +22,14 @@
      */
     public AddTagCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, CliSyntax.PREFIX_TAG);
```

Comment:
Similarly in this file, could you import the `CliSyntax` before with `import static nurseybook.logic.parser.CliSyntax.PREFIX_TAG;` instead?